build.gradle

```diff
@@ -0,0 +1,284 @@
+buildscript {
+  repositories {
+    maven {
+      url 'https://plugins.gradle.org/m2/'
+    }
+  }
+  dependencies {
+    classpath 'io.spring.gradle:dependency-management-plugin:1.0.4.RELEASE'
+    classpath 'com.github.jengelman.gradle.plugins:shadow:2.0.2'
+    classpath 'com.google.protobuf:protobuf-gradle-plugin:0.8.4'
+    classpath 'com.moowork.gradle:gradle-node-plugin:1.2.0'
+    classpath 'org.jfrog.buildinfo:build-info-extractor-gradle:4.6.2'
+    classpath 'org.sonarsource.scanner.gradle:sonarqube-gradle-plugin:2.6.2'
+  }
+}
+
+apply plugin: 'org.sonarqube'
+sonarqube {
+  properties {
+    property 'sonar.projectName', 'SonarQube'
```

Comment:
there's no native gradle field for project name? Can't it be set in build.gradle or settings.gradle?

Reply:
No, this is the name of the project (i.e `sonarqube`) and the artifact Id. There is nothing equivalent to the Maven property `project.name` with Gradle

Reply:
I understand, but can't we define a dedicated and shared variable, not specific to sonarqube task?